architectures

```diff
@@ -1,3 +1,4 @@
 bashbrew-arch   variants
 amd64     default,alpine,onbuild,slim,stretch,wheezy
 ppc64le default,onbuild,slim,stretch
+arm64  default
```

Comment:
my point is that you should change `arm64` to `arm64v8` in this file, then `generate-stackbrew-library.sh` would be able to generate the `architectures` information correctly.